Profiles - ZIB 2017/zib-LaboratoryTestResult-Observation.xml

```diff
@@ -47,8 +48,8 @@
-      <comment value="This profile is used both for the LaboratoryTestResult concept and the RelatedResult concept of HCIM LaboratoryTestResult. A LaboratoryTestResult can contain mutiple related RelatedResults. This is implemented using a parent Observation with related child Observations. See the code and related elements." />
+      <definition value="A laboratory result describes the result of a laboratory analysis.&#xD;&#xA;These are specimen-oriented tests as performed in laboratories such as Clinical Chemistry, Serology, Microbiology, etc.&#xD;&#xA;In addition to the results of tests with a singular result, this concept can also contain the results of more complex tests with multiple results or a ‘panel’." />
+      <comment value="This profile is used both for the HCIM LaboratoryTestResult root concept (NL-CM:13.1.1) and for the LaboratoryTest concept (NL-CM:13.1.3) within this HCIM. A LaboratoryTestResult can consist of just one LaboratoryTest or can be a panel of multiple LaboratoryTests. If there's just one LaboratoryTest, a single instance of this profile may be used to represent both concepts, but if there are multiple LaboratoryTests, there should be one parent instance representing the LaboratoryTestResult that uses `Observation.related` to link child instances representing individual LaboratoryTests.&#xD;&#xA;&#xD;&#xA;Note that the `Observation.related` field in is also used for a third case where there are multiple related LaboratoryTestResults (RelatedResult, NL-CM:13.1.33).&#xD;&#xA;&#xD;&#xA;The list below summarizes the different situations:&#xD;&#xA;1. Single LaboratoryTestResult with single LaboratoryTest:&#xD;&#xA;* single instance of this profile&#xD;&#xA;* `.code` represents the HCIM concept Test (NL-CM:13.1.8)&#xD;&#xA;2. Single LaboratoryTestResult with multiple LaboratoryTests&#xD;&#xA;* parent instance with child instances in `.related.target` (&#xD;&#xA;* `.code` of parent represents the HCIM concept PanelOrBattery (NL-CM:13.1.4)&#xD;&#xA;* `.code` of child represents the HCIM concept Test (NL-CM:13.1.8)&#xD;&#xA;* `.related.type` of parent set to &quot;has-member&quot;&#xD;&#xA;3. Multiple related LaboratoryTestResults&#xD;&#xA;* multiple instances representing the root concept&#xD;&#xA;* `.code` represents the HCIM concept Test (NL-CM:13.1.8)&#xD;&#xA;* `.related.target` references the related instances&#xD;&#xA;* `.related.type` set to whichever value is appropriate&#xD;&#xA;&#xD;&#xA;Also note that there is the additional profile [zib-LaboratoryTestResult-DiagnosticReport](http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult-DiagnosticReport), which is used for (textual) reports on individual (panels of) observations, like conclusions. The DiagnosticReport contains references to instances of this profile and can be used in all three the situations described above." />

```

Comment:
Perhaps it could be a good idea to start with the sentence: "A LaboratoryTestResult can consist of just one"

It is a very readable part while the first sentence is not. Could the first sentence be omitted? Or concluded at the end?

Small typo in: "Note that the `Observation.related` field **in is** also"

But in general: this is a big improvement!

Reply:
I think the concepts need to be introduced before using them in a sentence, but I'll try to reword it a bit.